values.yaml

```diff
@@ -316,6 +316,9 @@ global:
       gitSync:
         repository: quay.io/astronomer/ap-git-sync
         tag: 4.2.3-1
+      xcomm:
```

Comment:
Why is this xcomm instead of xcom? <https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/xcoms.html>